plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/ParamsUtils.java

```diff
@@ -12,6 +12,9 @@
 
 /**
  * Contains util methods for {@code *Params} classes.
+ *
+ * @author Mykola Morhun
+ * @author Alexander Garagatyi
  */
 public class ParamsUtils {
 
```

Comment:
up to you : Utility classes should not have public constructors https://dev.eclipse.org/sonar/rules/show/squid:S1118